src/libs/ReportUtils.js

```diff
@@ -1018,9 +1016,10 @@ function isUnread(report) {
         return false;
     }
 
-    const lastReadSequenceNumber = report.lastReadSequenceNumber || 0;
-    const maxSequenceNumber = report.maxSequenceNumber || 0;
-    return lastReadSequenceNumber < maxSequenceNumber;
+    // lastActionCreated and lastReadTime are both datetime strings and can be compared directly
+    const lastActionCreated = report.lastActionCreated || '';
+    const lastReadTime = report.lastReadTime || '';
+    return lastReadTime < lastActionCreated;
```

Comment:
Edge case: If a chat report is empty (has no report actions except the `CREATED` action) then `lastReadTime` wouldn't be set and we fallback to an empty string and `'' < lastActionCreated` will be `true` making an empty chat unread (https://github.com/Expensify/App/issues/38368).